docs/category.yaml

```
@@ -0,0 +1,222 @@
+                  createdAt:
+                    type: string
+                    format: date-time
+                    description: The time the category was created
+                  updatedAt:
+                    type: string
+                    format: date-time
+                    description: The time the category was last updated
+        '400':
+          description: Invalid category ID format
+        '404':
+          description: Category not found
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)